xarray/core/missing.py

```diff
@@ -734,20 +734,34 @@ def interp_func(var, x, new_x, method, kwargs):
         else:
             dtype = var.dtype
 
-        return da.blockwise(
-            _dask_aware_interpnd,
-            out_ind,
-            *args,
-            interp_func=func,
-            interp_kwargs=kwargs,
-            localize=localize,
-            concatenate=True,
-            dtype=dtype,
-            new_axes=new_axes,
-            # TODO: uncomment when min dask version is > 2.15
-            # meta=var._meta,
-            align_arrays=False,
-        )
+        if dask_version < "2020.12":
+            # Remove this whenever the minimum requirement for dask is 2020.12:
```

Comment:
How about: 
```
if dask_version < "2020.12":
	meta = None
else:
	meta = var._meta

return da.blockwise(
	...,
	meta=meta
)
```

Reply:
Sure, I just thought that required more brainpower whenever someone cleans this up.